recipes/ozzy-pic/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python_min:
```

Comment:
This is fine but I prefer the line

```
{% set python_min = python_min|default("3.8") %}
```

in the recipe b/c it is easier to remember when to update it.